### PR TITLE
DTSERWFOUR-98 Enable Spanish Translations for app and ui sdk

### PR DESCRIPTION
**Pull request by pdamodarannair**

### CODE REVIEW

File: transfermethodui/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -76,5 +76,5 @@
     <string name="mobileTransferMethodsHeader">Transfer methods</string>
 
     <string name="mobileRemoveEAconfirm">Remove transfer method</string>
-    <string name="prepaidCardManagementInfo">"Log in using a web browser to manage your card"</string>
+    <string name="prepaidCardManagementInfo"  translatable="false">"Log in using a web browser to manage your card"</string>
```
[the remark this answers is not in the document]
Comment (pdamodarannair) [reply transfermethodui/src/main/res/values/strings.xml:79]: Resolved

**Review by shanavas123 — DISMISSED** [review]

**Review by shanavas123 — DISMISSED** [review]

**Review by rajkumar-balakrishnasamy — DISMISSED** [review]

File: commonui/src/main/res/values-es/strings.xml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    <string name="cash_pickup_mg">MoneyGram</string>
+
+    <string name="bank_account_font_icon">\ue900</string>
+    <string name="bank_card_font_icon">\ue903</string>
+    <string name="wire_account_font_icon">\ue90a</string>
+    <string name="paper_check_font_icon">\ue902</string>
+    <string name="prepaid_card_font_icon">\ue901</string>
+    <string name="paypal_account_font_icon">\ue906</string>
+    <string name="venmo_account_font_icon">\ue909</string>
+    <string name="cash_pickup_mg_font_icon">\ue905</string>
```
Comment (vwagh-hw) [inline commonui/src/main/res/values-es/strings.xml:26]: Nothing to translate here for icons, remove it from translated files

Comment (pdamodarannair) [reply commonui/src/main/res/values-es/strings.xml]: Removed

File: receiptui/src/main/res/values-es/strings.xml [hunk 3]
```diff
@@ -0,0 +1,166 @@
...
+    <string name="concat_string_view_format">%s%s %s</string>
+    <string name="concat_date_string_view_format">%s %s</string>
+
+    <string name="mobileNoTransactions">No hay actividad para mostrar.</string>
+    <string name="title_transfer_funds">Transferir fondos</string>
+    <string name="font_icon_placeholder">\ue908</string>
+    <string name="text_placeholder">Marcador de posición</string>
+
+    <!-- entry types -->
+    <string name="debit">\ue908</string>
```
Comment (shanavas123) [inline receiptui/src/main/res/values-es/strings.xml:29]: nothing to translate

File: receiptui/src/main/res/values-es/strings.xml [hunk 4]
```diff
@@ -0,0 +1,166 @@
...
+    <string name="credit">\ue907</string>
```
Comment (shanavas123) [inline receiptui/src/main/res/values-es/strings.xml:30]: nothing to translate

File: receiptui/src/main/res/values-es/strings.xml [hunk 5]
```diff
@@ -0,0 +1,166 @@
...
+    <string name="card_activation_fee">Tarifa de activación de tarjeta</string>
+    <string name="card_activation_fee_waiver">Exención de la tarifa de activación de tarjeta</string>
+    <string name="card_fee">Tarifa de la tarjeta</string>
+    <string name="manual_transfer_to_prepaid_card">Carga de la tarjeta</string>
+    <string name="prepaid_card_balance_inquiry_fee">Tarifa de consulta de saldo</string>
+    <string name="prepaid_card_cash_advance">Anticipo de efectivo</string>
+    <string name="prepaid_card_disputed_charge_refund">Devolución de cargos impugnados</string>
+    <string name="prepaid_card_dispute_deposit">Depósito de tarjeta impugnada</string>
+    <string name="prepaid_card_domestic_cash_withdrawal_fee">Retirada del cajero automático</string>
+    <string name="prepaid_card_exchange_rate_difference">PREPAID_CARD_EXCHANGE_RATE_DIFFERENCE
```
Comment (shanavas123) [inline receiptui/src/main/res/values-es/strings.xml:73]: check string in English

File: receiptui/src/main/res/values-es/strings.xml (hunk 3 above)
Comment (pdamodarannair) [reply receiptui/src/main/res/values-es/strings.xml]: Removed

File: receiptui/src/main/res/values-es/strings.xml (hunk 5 above)
Comment (pdamodarannair) [reply receiptui/src/main/res/values-es/strings.xml:67]: Received the same keyword from Husayn

**Review by shanavas123 — APPROVED** [review]

**Review by vwagh-hw — APPROVED** [review]

**Pull request merged**